pulsar-client/src/test/java/org/apache/pulsar/client/impl/schema/JSONSchemaTest.java

```diff
@@ -368,4 +367,33 @@ public void testEncodeAndDecodeObject() throws JsonProcessingException {
         PC roundtrippedPc = jsonSchema.decode(encoded);
         assertEquals(roundtrippedPc, pc);
     }
-}
+
+    @Test
+    public void testDateAndTimestamp() {
```

Comment:
why this test is named "testDateAndTimestamp" ?
isn't it about testing the GenericRecordBuilder ?

Reply:
Yes, that was the origin test name I duplicated ! I rename it to testJsonGenericRecordBuilder